packages/esm-patient-common-lib/src/orders/useOrders.ts

```diff
@@ -15,7 +20,10 @@ export function usePatientOrders(patientUuid: string, status: 'ACTIVE' | 'any',
   );
 
   const mutateOrders = useCallback(
-    () => mutate((key) => typeof key === 'string' && key.startsWith(`/ws/rest/v1/order?patient=${patientUuid}`)),
+    () =>
+      mutate((key) => {
```

Comment:
`mutate` seems like it's no longer defined here?

Reply:
The linter just moved it to the next line